reports/California.md

```diff
@@ -49,6 +49,22 @@ id: ca-compton-1
 * https://ktla.com/news/local-news/violent-arrest-caught-on-camera-in-compton/
 * [Street View](https://www.google.com/maps/@33.9038142,-118.2141026,3a,75y,255.55h,58.86t/data=!3m6!1e1!3m4!1sDJvjgZKDPB56Yw1A5Ovb6A!2e0!7i16384!8i8192)
 
+### Sheriff's Deputies fire rubber bullets, tear-gas at protesters | June 21
+
+Several thousand protesters marched on Father's Day from the site of LASD shooting of Andrés Guardado on June 18 to the Los Angeles Sheriff's station in Compton. Andrés' father addressed the crowd in English and in Spanish. Deputies in riot gear pointed guns at protesters who chanted "put down your guns." A Sheriff Department helicopter flew overhead urging local residents who had joined the march to leave and go home so that they would not be teargassed when outside agitators were dealt with. Black protesters and protest leaders shared hugs with Guardado's family to show solidarity between the Black and Latino communities. As a few mothers of children slain by LAPD or LASD told their stories, a group of protesters split off to confront a second band of Deputies in riot gear. One protester yelled "F*** the police!" The deputies opened fire with rubber bullets, tear-gas, and flash bangs.
```

Comment:
I believe this is the same incident as `ca-losangeles-21`. See [here](https://github.com/2020PB/police-brutality/blob/master/reports/California.md#police-tear-gas-peaceful-protest--june-21st) We haven't been super strict about LA vs Compton because it's the same PD. This is great information though, so if you want to move the LA incident here or supplement the LA incident with this information

Reply:
You're right - it is the same incident. Compton and LA have different police departments. In LA there is LA Police Dept (LAPD) for the city of Los Angeles, and LA Sheriffs Department for the county and for many of the other cities in the county. Compton is LASD. LASD also runs the jails and are particularly violent. Two Sheriffs ago and his under sheriff currently both in federal prison on brutality convictions. Will bring ca-losangeles-21 into this one.

Reply:
This is also the same incident as ca-losangeles-22

Reply:
Thanks for the clarification! If you see similar incidents that need to be shifted around, please feel free to open new PRs. I'm on the east coast, so these differences are new to me.

Reply:
ca-losangeles-21/22 both folded into compton-2; links and descriptions added.